lib/services/ucs-obm-service.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di'),
+    util = require('util');
+
+module.exports = UcsObmServiceFactory;
+
+di.annotate(UcsObmServiceFactory, 
+    new di.Provide('ucs-obm-service')
```

Comment:
Add the new obm service into schema https://github.com/RackHD/on-tasks/blob/master/lib/task-data/base-tasks/obm.js#L30, otherwise this obm will not pass the basic validation.